setup.py

```diff
@@ -24,7 +24,9 @@
         'nmigen-yosys',
         'edalize==0.2.3',
         'fusesoc==1.11.0',
-        'nmigen @ git+https://github.com/nmigen/nmigen',
+        'nmigen @ git+https://github.com/nmigen/nmigen@'
+        'db5a981f438df7ec4f7f5b2224b5f01a36d79a44',
```

Comment:
This looks weird, I guess the line break is here because of flake? Could we shorten the hash to lets say 12 chars and merge these lines?

Reply:
This is a single line now :+1: